src/pages/sponsor-and-support.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import Link from 'gatsby-link'
+import get from 'lodash/get'
+import Helmet from 'react-helmet'
+
+import HeaderGeneric from '../components/HeaderGeneric'
+
+class SponsorAndSupport extends React.Component {
+  render() {
+
+    return (
+      <div>
+        <Helmet title={get(this, 'props.data.site.siteMetadata.title')} />
+
+        <HeaderGeneric pageTitle='Sponsor and Support' pageSummary='✨ Build your community ✨' />
+        <div id="main">
+          <section id="content" className="main">
+            <h2>Sponsorship Package</h2>
+            <p>Sponsorships made to Gastown Gang will grow all the events under its umbrella.</p>
+
+            <h2>Types of sponsorship</h2>
+            <p>You or your organization can be either a <em>venue</em> or a <em>cash</em> sponsor.</p>
+
+            <h3>Venue</h3>
+            <p>Venue sponsors may chose to host one or several of Gastown Gang-supported events.</p>
+
+            <h4>What you get:</h4>
+            <ul>
+              <li>A personalized mention by our emcee at each event.</li>
+              <li>Two minutes to speak at the start of the event. One of your employees or developer relations person should speak — no recruiters! You may prepare something for us to read. You may include job opportunities during your two minutes, but please don’t recruit audience members during socialization time.</li>
```

Comment:
I'd specify they *may* have one of their employees talk about the company--we may get companies interested in sponsoring but don't have local employees (yet?).